sale_order_general_discount_fixed/tests/test_sale_order_general_discount_fixed.py

```
@@ -0,0 +1,287 @@
+        form.partner_id = self.partner
+        with form.order_line.new() as line:
+            line.product_id = self.product_100
+            line.product_uom_qty = 1
+        with form.order_line.new() as line:
+            line.product_id = self.product_300
+            line.product_uom_qty = 1
+        form.general_discount_fixed = 40.0
+        order = form.save()
+
+        line_to_remove = order.order_line.filtered(
+            lambda l: not l.display_type and l.price_unit == 100.0
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)